<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMG Recorder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>492</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>492</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="textBrowserSerial">
      <property name="maximumSize">
       <size>
        <width>273</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="PlotWidget" name="graphWidget" native="true"/>
    </item>
    <item row="1" column="1" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="comboBoxPorts">
        <property name="minimumSize">
         <size>
          <width>191</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="text">
         <string>Verbinden</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDisconnect">
        <property name="text">
         <string>Trennen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelInfotext">
        <property name="font">
         <font>
          <pointsize>23</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Recorder Infotext</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStopRecord">
        <property name="text">
         <string>Aufnahme stoppen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRecord">
        <property name="text">
         <string>Aufnahme starten</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="labelPicture">
      <property name="text">
       <string>Picture</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Verbinden</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRecord</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1146</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDisconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnectSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStopRecord</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1014</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>925</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectSerial()</slot>
  <slot>startRecord()</slot>
  <slot>disconnectSerial()</slot>
  <slot>stopRecord()</slot>
 </slots>
</ui>
